pdf/pdf.py

```diff
@@ -77,6 +77,12 @@ def student_view(self, context=None):
         }
         html = self.render_template('static/html/pdf_view.html', context)
 
+        event_type = 'edx.pdf.loaded'
+        event_data = {
+            'url': self.url,
```

Comment:
Because `url` is always printed but `source_url` is only downloadable. It's not loaded. 
But I can add it.